style.css

```diff
@@ -2,7 +2,7 @@
 Theme Name: UCF.edu theme
 Author: UCF Web Communications
 Github Theme URI: https://github.com/UCF/Main-Site-Theme
-Version: v3.23.1
+Version: v3.23.2
```

Comment:
Please do not update the version in pull requests. This can be updated as a single commit right before tagging.

There's a certain amount of "ritual" around updating the version before tagging, and around actually tagging and creating a release. Both actions are an indication that we're declaring code finished and ready to release. Being very intentionally about when we do these things are what prevent us from making mistakes when we're rushing to get things done. It seems silly, but it matter.